### DISCUSSION TITLE
Next/previous chapter buttons?

### DISCUSSION THREAD

**Discussion by HubKing** [tomasklaen/uosc#717] [Q&A] [answered]
Current next/prev buttons (<, >) moves to next/previous file. Can I have buttons for next/previous chapters?

**Comment by tomasklaen** [comment]
You can customize controls and have any mpv command you want in there. The syntax for defining control buttons is documented in the `uosc.conf` file right above `controls` option.

**Comment by christoph-heinrich** [comment, marked as answer]
I have those in my controls
```
<has_chapter>command:arrow_back_ios:no-osd add chapter -1?Previous,<has_chapter>chapters,<has_chapter>command:arrow_forward_ios:no-osd add chapter 1?Next,
```
